batch/indexer_Token_Detail.py

```diff
@@ -120,7 +120,6 @@ def __sync(self, local_session: Session):
                     token_detail = token_detail_obj.to_model()
                     token_detail.created = datetime.utcnow()
```

Comment:
I did not modify or remove this line because I thought that IDXToken meant cached data.
(The column `created` means when cache data is created I think)